VideoQuickStart/ViewController.swift

```diff
@@ -129,9 +142,18 @@ class ViewController: UIViewController {
         let connectOptions = TVIConnectOptions.init(token: accessToken) { (builder) in
             
             // Use the local media that we prepared earlier.
-            builder.audioTracks = self.localAudioTrack != nil ? [self.localAudioTrack!] : [TVILocalAudioTrack]()
-            builder.videoTracks = self.localVideoTrack != nil ? [self.localVideoTrack!] : [TVILocalVideoTrack]()
-            
+            if let localAudioTrack = self.localAudioTrack {
+                builder.audioTracks = [localAudioTrack]
+            }
+
+            if let localVideoTrack = self.localVideoTrack {
+                builder.videoTracks = [localVideoTrack]
+            }
+
+            if let localDataTrack = self.localDataTrack {
+                builder.dataTracks = [localDataTrack]
+            }
```

Comment:
What if this is a Group Room? Should we have a boolean to disable this functionality by default, and allow the developer to enable it when they are in a P2P Room? This would be temporary until preview6 is available.

Reply:
How do we know we are in a group room?

Reply:
Unfortunately we as an SDK don't.

Reply:
Right, I wrote that before I fully digested your comment.

Reply:
tried group room, since existing audio+video functionalities are working in group rooms, we will move ahead with readme note about Group Room.